specification/data-resource-semantic-conventions.md

```diff
@@ -57,17 +57,24 @@ namespace = Company
 service.name = Shop.shoppingcart
 ```
 
-## Library
+## Telemetry SDK
 
-**type:** `library`
+**type:** `telemetry.sdk`
 
-**Description:** Telemetry library information.
+**Description:** The SDK used to produce telemetry data.
+
+In OpenTelemetry, this is the OpenTelemetry SDK implementation being used.  
+If the default SDK provided by the OpenTelemetry project is used, `telemetry.sdk.name` MUST
+be set to `opentelemetry-sdk`.  
+If another SDK, like a fork or a vendor-provided SDK is used, `telemetry.sdk.name` MUST be set
+to the fully-qualified class or module name of that SDK's main entry point, depending on the language.
```

Comment:
What if it is made up of multiple modules? What's the main entry point if an SDK is slit up in e.g. tracing and metrics?

Reply:
I left it rather vague on purpose since it's difficult to find a clear definition that suits all potential vendor implementations. I thought of the OpenTelemetry SDK where (pretty much?) all implementations have a global OpenTelemetry object as an entrypoint which exposes the global tracer and meter providers. I'm open to better suggestions of course!

Reply:
The only option would be making it a list. But this would then require keeping multiple lists (version, tech) in sync which is ugly. Let's leave it like this for now.